tests/ui/macros/cfg_select.rs

```diff
@@ -25,3 +25,6 @@ cfg_select! {
     //~^ ERROR none of the rules in this `cfg_select` evaluated to true
     false => {}
 }
+
+cfg_select! {}
+//~^ ERROR none of the rules in this `cfg_select` evaluated to true
```

Comment:
It's not necessarily a matter for this PR, but we should probably update this to refer to predicates rather than rules.

Reply:
I think it's easier to just have the terminology be consistent, so I replaced "rule" with "predicate" everywhere.